lib/main/server.js

```diff
@@ -0,0 +1,50 @@
+var http = require('http');
+var url = require('url');
+var Scrapper = require('./scrapper.js');
+
+function Server() {
+
+  this.start = function() {
+    http.createServer(function(request, response) {
+      var urlParams = url.parse(request.url, true);
+      if (request.method === 'GET' && urlParams.pathname === '/I/want/title/') {
+        var addresses = (typeof(urlParams.query.address) == 'string') ? [urlParams.query.address] : urlParams.query.address;
+        var titles = [];
+        var body = [];
+
+        function scrapTitle(index, callback) {
```

Comment:
Function definition over here reduces the readability of code. I believe it is good to move it downwards. Yet better is to move outside this function all together.